lib/tasks/cap_cutover.rake

```diff
@@ -102,363 +33,13 @@ namespace :cap_cutover do
-  task build_missing_pubs: :environment do
-    include ActionView::Helpers::DateHelper
-    start_time = Time.zone.now
-    total_running_count = 0
-    @cap_import_logger = Logger.new(Rails.root.join('log', 'cap_build_pubs.log'))
-    @cap_import_logger.datetime_format = '%Y-%m-%d %H:%M:%S'
-    @cap_import_logger.formatter = proc { |severity, datetime, _progname, msg|
-      "#{severity} #{datetime}: #{msg}\n"
-    }
-    @cap_import_logger.info 'Started cap build pub process ' + Time.zone.now.to_s
-
-    @no_pubmed_rec = []
-    Publication.find_each do |pub|
-      total_running_count += 1
-      build_pub_from_missing_sw_and_pubmed(pub)
-      GC.start if total_running_count % 5000 == 0
-
-      if total_running_count % 1000 == 0
-        Rails.logger.debug "#{total_running_count} in #{distance_of_time_in_words_to_now(start_time, true)}"
-        @cap_import_logger.info total_running_count.to_s + ' in ' + distance_of_time_in_words_to_now(start_time, true)
-      end
-    end
-    @cap_import_logger.info "No pubmed record for #{@no_pubmed_rec.size} items:"
-    @no_pubmed_rec.each { |id| @cap_import_logger.info id.to_s }
-    @cap_import_logger.info 'Finished build.' + Time.zone.now.to_s
-    @cap_import_logger.info total_running_count.to_s + 'records were processed in ' + distance_of_time_in_words_to_now(start_time, true)
-  end
-
-  desc 'utility to rebuild pub hashes from sciencewire and pubmed sources for all pubs'
-  task :build_delta_pubs, [:file_location] => :environment do |_t, args|
-    include ActionView::Helpers::DateHelper
-    start_time = Time.zone.now
-    total_running_count = 0
-    @cap_import_logger = Logger.new(Rails.root.join('log', 'cap_build_delta_pubs.log'))
-    @cap_import_logger.info 'Started cap build pub process ' + Time.zone.now.to_s
-
-    CSV.foreach(args.file_location, headers: true, header_converters: :symbol) do |row|
-      begin
-        pub = Publication.where(pmid: row[:pubmed_id]).first
-        if pub.nil?
-          @cap_import_logger.warn "Could not find pub for pmid #{row[:pubmed_id]}"
-          next
-        end
-        total_running_count += 1
-        build_pub_from_sw_and_pubmed(pub)
-        GC.start if total_running_count % 5000 == 0
-
-        if total_running_count % 1000 == 0
-          Rails.logger.debug "#{total_running_count} in #{distance_of_time_in_words_to_now(start_time, true)}"
-          @cap_import_logger.info total_running_count.to_s + ' in ' + distance_of_time_in_words_to_now(start_time, true)
-        end
-      rescue => e
-        @cap_import_logger.error "Problem with #{row.inspect}"
-        @cap_import_logger.error e.inspect << "\n" << e.backtrace.join("\n")
-      end
-    end
-    @cap_import_logger.info 'Finished build.' + Time.zone.now.to_s
-    @cap_import_logger.info total_running_count.to_s + 'records were processed in ' + distance_of_time_in_words_to_now(start_time, true)
-  end
```

Comment:
OK, `build_delta_pubs` noted for deprecation in https://github.com/sul-dlss/sul_pub/issues/356#issuecomment-341251365